src/main/scala/org/broadinstitute/dsde/firecloud/EntityService.scala

```diff
@@ -211,6 +212,7 @@ class EntityService(rawlsDAO: RawlsDAO, importServiceDAO: ImportServiceDAO, mode
     response map { response =>
       response.status match {
         case NoContent =>
+          response.discardEntityBytes()
```

Comment:
what does discardEntityBytes do?

Reply:
the comments in the akka-http code explain it _much_ better than i could, so here they are: 

```     
* Discards the entities data bytes by running the `dataBytes` Source contained by the `entity` of this HTTP message.
     *
     * Note: It is crucial that entities are either discarded, or consumed by running the underlying [[akka.stream.scaladsl.Source]]
     * as otherwise the lack of consuming of the data will trigger back-pressure to the underlying TCP connection
     * (as designed), however possibly leading to an idle-timeout that will close the connection, instead of
     * just having ignored the data.
```